jsk_data/node_scripts/data_collection_server.py

```diff
@@ -91,6 +93,10 @@ def __init__(self):
         self.timestamp_save_dir = rospy.get_param('~timestamp_save_dir', True)
         self.wait_timer = rospy.get_param('~wait_timer', False)
         self.wait_save_request = rospy.get_param('~wait_save_request', False)
+        self.rosbag = rospy.get_param('~rosbag', False)
+        if self.rosbag:
+            self.rosbag_process = None
+            self.rosbag_topics = rospy.get_param('~rosbag_topics', [])
```

Comment:
Hou about adding ```rospy.get_param('~rosbag_options', [])```?

Reply:
Thank you for your review.
I don't understand it well, but is it to integrate the rosbag type params (like `rosbag_topics` and `rosbag_prefix`) into `rosbag_options` param?

Reply:
We can pass an option to the rosbag record, so it would be nice if the user could pass an option.

Reply:
I see.
I'm wondering if options related to topics and file names such as -a and -o should be supported, but I made it possible to specify the option of rosbag record from `rosbag_options` param.

Reply:
`rosbag record` has a lot of options, and other jsk's programs use such that options. So, it's good features. Thanks.
http://wiki.ros.org/rosbag/Commandline#record
https://github.com/jsk-ros-pkg/jsk_common/blob/master/jsk_data/launch/rosbag_always_run_rosbag.launch#L7